.travis/test_react_native.sh

```diff
@@ -0,0 +1,5 @@
+cd extensions/applicationinsights-react-native
```

Comment:
what about adding `test_angular.sh` like so: 
`
cd extensions/applicationinsights-angular
npm install
npm run build
npm run test
npm run lint
`
we can also adding it later after angular plugin is published.

Reply:
added `test_angular.sh`